src/gameClasses/ClientNetworkEvents.js

```diff
@@ -574,9 +574,15 @@ var ClientNetworkEvents = {
 				break;
 			case "playSoundForPlayer":
 				var sound = taro.game.data.sound[data.sound];
+				var volume = taro.game.data.sound[data.volume];
+
 				if (sound) {
 					var unit = taro.client.myPlayer && taro.client.myPlayer.getSelectedUnit();
-					taro.sound.playSound(sound, (unit && unit._translate) || null, data.sound);
+					if (volume) {
+						taro.sound.playSound(sound, (unit && unit._translate) || null, data.sound, volume);
+					} else {
+						taro.sound.playSound(sound, (unit && unit._translate) || null, data.sound, 100);
+					}
```

Comment:
Wouldn't that be an approach that's easier to understand?

```suggestion
					taro.sound.playSound(sound, (unit && unit._translate) || null, data.sound, volume ?? 100);
```

depending on which version/variant/style of javascript you're going for: `||` works as well

Reply:
Either way works I guess I just did it that way since I'm new to js